imagebuildah/stage_executor.go

```diff
@@ -907,10 +907,10 @@ func (s *StageExecutor) Execute(ctx context.Context, base string) (imgID string,
 			command := strings.ToUpper(step.Command)
 			// chmod, chown and from flags should have an '=' sign, '--chmod=', '--chown=' or '--from='
 			if command == "COPY" && (flag == "--chmod" || flag == "--chown" || flag == "--from") {
-				return "", nil, errors.Errorf("COPY only supports the --chmod=<permissions> --chown=<uid:gid> and the --from=<image|stage> flags")
+				return "", nil, errors.Errorf("COPY only supports the --keep-ownership --chmod=<permissions> --chown=<uid:gid> and the --from=<image|stage> flags")
```

Comment:
```suggestion
				return "", nil, errors.Errorf("COPY only supports the --keep-ownership, --chmod=<permissions> --chown=<uid:gid> and the --from=<image|stage> flags")
```

Reply:
Should I also add also a Comma after `--chmod=<permissions>` I was thinking about that, but thought I keep the current format.

Reply:
>Should I also add also a Comma after --chmod=<permissions>

Yes, and also after `--chmod=<uid:gid>`. The term is `Oxford comma` and this is an instance where it can help clarity. You can also remove the superfluous `the` and not-really-helpful `=<blah>`, simplifying to:
```
...the --chmod, --chown, --from, and --keep-ownership flags
```
Also, I realize this is a WIP, but you're going to need to squash your commits (to 2, I think) at some point. There's no need nor desire for a stack of "address review feedback" commits.

Thanks again for contributing!

Reply:
Thanks for your feedback, will change all you mentioned points later.